sdk/cosmos/azure-cosmos/azure/cosmos/_cosmos_client_connection.py

```diff
@@ -26,19 +26,18 @@
 """
 from typing import Dict, Any, Optional
 import six
-import requests
-from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry  # pylint: disable=import-error
```

Comment:
That's too risky on Ubuntu machines, that might blow up.
Being you depend en azure-core, that requires request 2.18 at least, you know for sure urllib3 is packaged *outside* of requests whatever platform its used.
So I would do:
```python
from urllib3.util.retry import Retry
```